models/public/mobilenet-yolo-syg/model.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright © 2018-2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This is a Keras\* version of `mobilenet-yolov4` model, one of the YOLO group of models
+  designed to perform real-time object detection mainly to vehicles.
+  The weights are pretrained by BDD100k and retrained by our own dataset.
+  For details see repository <https://github.com/ermubuzhiming/omz-test/>,
+  paper of MobileNetV2<https://arxiv.org/abs/1801.04381> and YOLOv4<https://arxiv.org/abs/2004.10934>
+task_type: detection
+
```

Comment:
```suggestion
```

Reply:
sorry for missing updating the repo link ,I update it in the model.yml.